app/package.json

```diff
@@ -152,5 +152,8 @@
     "prettier": "^1.18.2",
     "react-app-rewired": "^2.1.6",
     "rimraf": "^3.0.2"
-  }
+  },
+  "resolutions": {
+    "pull-ws": "3.3.2"
```

Comment:
@pRoy24 just wondering what did this do to fix it?

Reply:
the resolutions directive forces yarn to resolve sub-dependancies using the specified version instead of using the version specified in the package json of the dependancy being installed.
https://classic.yarnpkg.com/en/docs/selective-version-resolutions/